<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="SearchEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="NameButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AddressButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="ShowLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>351</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;WelcomeÔºÅ&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Please follow this steps to use the system:&lt;/p&gt;&lt;p&gt;1. Choose the method of searching. If you &lt;/p&gt;&lt;p&gt;want to search by the name of restaurant, &lt;/p&gt;&lt;p&gt;please click the &amp;quot;Name&amp;quot; button. If you want &lt;/p&gt;&lt;p&gt;to search by the address of restaurant, &lt;/p&gt;&lt;p&gt;please click the &amp;quot;Address&amp;quot; button.&lt;/p&gt;&lt;p&gt;2. Type your key words on the line.&lt;/p&gt;&lt;p&gt;3. Click the &amp;quot;Search&amp;quot; button.&lt;/p&gt;&lt;p&gt;4. Click the &amp;quot;Map&amp;quot; button to see the result &lt;/p&gt;&lt;p&gt;on a map.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="ListLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>561</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MapButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProvidence_Food_Map">
    <property name="title">
     <string>Providence Food Map</string>
    </property>
   </widget>
   <addaction name="menuProvidence_Food_Map"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>enableSearchButton(QString)</slot>
  <slot>search()</slot>
 </slots>
</ui>
